scripts/assets/file_content_validation_checks.textproto

```diff
@@ -408,3 +408,13 @@ file_content_checks {
   exempted_file_name: "app/src/main/java/org/oppia/android/app/application/AbstractOppiaApplication.kt"
   exempted_file_patterns: ".+?Test\\.kt"
 }
+file_content_checks {
+  file_path_regex: ".+?.kt"
+  prohibited_content_regex: ".+?\\.(post|postDelayed)\\(.+?"
+  failure_message: "Using post() and postDelayed() method is prohibited as it can lead to NullPointerException errors. Use doOnPreDraw or doOnLayout instead. Checkout this resource for a better understanding of the issue: https://betterprogramming.pub/stop-using-post-postdelayed-in-your-android-views-9d1c8eeaadf2"
+  exempted_file_name: "app/src/main/java/org/oppia/android/app/spotlight/SpotlightFragment.kt"
+  exempted_file_name: "app/src/main/java/org/oppia/android/app/topic/info/TopicInfoFragmentPresenter.kt"
+  exempted_file_name: "app/src/main/java/org/oppia/android/app/drawer/NavigationDrawerFragmentPresenter.kt"
+  exempted_file_name: "app/src/main/java/org/oppia/android/app/player/state/StateFragmentPresenter.kt"
+  exempted_file_patterns: ".+?Test\\.kt"
```

Comment:
Do any tests actually use post/postDelayed? I don't think we want to exempt this there, either, as tests probably shouldn't be using these functions either.

Reply:
Please make sure to respond to this @kkmurerwa.

Reply:
I have removed the exemption for all tests and only added the `RegexPatternValidationCheckTest` since it contains an instance of post/postDelayed in the prohibited content string.